util/list.go

```diff
@@ -14,68 +14,71 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
 
-package ledger
+package util
 
-// persistedKVDataList represents a doubly linked list.
-// must initiate with newPersistedKVList.
-type persistedKVDataList struct {
-	root     persistedKVDataListNode  // sentinel list element, only &root, root.prev, and root.next are used
-	freeList *persistedKVDataListNode // preallocated nodes location
+// List represents a doubly linked.
```

Comment:
why removing the last word, "list"

Reply:
I hate that github shows four lines when I highlight one. I'm talking about the last word of the comment.

Reply:
A mistake, will fix.